cmd/clusterctl/examples/openstack/generate-yaml.sh

```diff
@@ -7,7 +7,7 @@ print_help()
   echo "$SCRIPT - generates a provider-configs.yaml file"
   echo ""
   echo "Usage:"
-  echo "$SCRIPT [options] <path/to/clouds.yaml> <cloud> <provider os: [centos,ubuntu,coreos]>"
+  echo "$SCRIPT [options] <path/to/clouds.yaml> <cloud> <provider os: [centos,ubuntu,coreos]> [output folder]"
```

Comment:
s/[output folder]/\<output folder\>

Reply:
the output folder is optional.... in my memory<> means mandatory and [] means optional?

Reply:
Ah, I see, thanks.

/lgtm